lib/batch/institutional_nucats_template.rb

```diff
@@ -0,0 +1,40 @@
+# https://github.com/galterlibrary/digital-repository/issues/691
```

Comment:
A more explicit title will help explain what this is doing: `create_blank_nucats_subcollections.rb`